tests/test_operational_limits.py

```diff
@@ -0,0 +1,220 @@
+"""A simple test of the operational limits plugin activity."""
+# -------------------------------------------------------------------------------------!
+# Import dependencies
+import datetime
+import os
+
+import dateutil
+import numpy as np
+import pandas as pd
+import simpy
+
+import openclsim.appendix as appendix
+import openclsim.model as model
+import openclsim.plugins as plugins
+
+from .test_utils import assert_log
+
+
+# -------------------------------------------------------------------------------------!
+def exec():
```

Comment:
bit of an unfortunate name. the exec method already exists.

```python
exec("ls")
```


if you call it test_opperational limit() then pytest will find it automaticly